homework-2/program2.java

```diff
@@ -0,0 +1,38 @@
+import java.util.*;
+public class program2
+{
+
+    public static void main(String args[])
+    {
+        String str;
+        Scanner sc = new Scanner(System.in);
+        HashMap <Character,Integer> map = new HashMap<Character,Integer>();
```

Comment:
Use a name for the map that indicates the mapping like frequencyOfChar or charToFrequency